styles.scss

```diff
@@ -1,7 +1,98 @@
+// FONT
+@import url("https://fonts.googleapis.com/css2?family=Fredoka+One&display=swap");
+
+//VARIABLES
+@import "./variables/mixins";
+@import "./variables/colors";
+
 html,
 body {
   margin: 0;
   padding: 0;
   font-size: 10px;
-  font-family: "Courier New", Courier, monospace;
+  font-family: "Fredoka One", cursive;
+  overflow: hidden;
+}
+
+.main {
+  display: flex;
+  align-items: center;
+  flex-direction: column;

```

Comment:
This can be a mixin: https://gist.github.com/nathan-hyan/851ef2094e5ba7c3b2b1e50567c0a2a8

Reply:
Looks so much better!